apps/prairielearn/public/stylesheets/colors.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../../../node_modules/bootstrap/scss/functions';
+@import '../../../../node_modules/bootstrap/scss/variables';
+@import '../../../../node_modules/bootstrap/scss/mixins';
+
+// We reuse Bootstrap's `button-variant` mixin to generate appropriate hover
+// styles. Crucially, this will ensure that appropriate color contrast is
+// maintained.
+//
+// As with normal buttons (see `local.css`), we also include an outline with
+// `:focus-visible` to ensure that the focus state has sufficient contrast,
+// no matter the background color on which the button is situated.
+//
+// Install `sass` with `npm install -g sass`.
+// Run `sass --sourcemap=none colors.scss colors.css` to compile this file.
```

Comment:
Should this be run/checked on CI?

Reply:
This file should more or less never change, so I didn't bother to do that. If/when we want that, we should integrate the SASS compiler into `@prairielearn/compiled-assets` so that we can put this in `apps/prairielearn/assets/stylesheets` and avoid another bespoke asset compilation thing. If you feel strongly about it, I can do that as part of this PR.

Reply:
That sounds reasonable.